lib/agora/join_active_stream.dart

```diff
@@ -0,0 +1,196 @@
+import 'package:flutter/material.dart';
+import 'package:thc/agora/active_stream.dart';
+import 'package:thc/agora/broadcasting/broadcast_stream.dart';
+import 'package:thc/credentials/credentials.dart';
+import 'package:thc/home/schedule/src/all_scheduled_streams.dart';
+import 'package:thc/start/src/login_fields.dart';
+import 'package:cloud_functions/cloud_functions.dart';
+import 'package:flutter/material.dart';
+import 'package:thc/home/schedule/scheduled_stream_card.dart';
+import 'package:thc/home/schedule/src/all_scheduled_streams.dart';
+
+class JoinActiveStream extends StatefulWidget {
+  const JoinActiveStream({super.key});
+
+  @override
+  State<JoinActiveStream> createState() => _JoinActiveStreamState();
+}
+
+class _JoinActiveStreamState extends State<JoinActiveStream> {
+  late final FocusNode _unfocusNode;
```

Comment:
This one doesn't need to be `late`:

```suggestion
  final FocusNode _unfocusNode = FocusNode();
```